docs/getting_started/installation.rst

```diff
@@ -124,6 +124,14 @@ or explicitly:
 
   conda install -c conda-forge modin-ray modin-dask modin-omnisci
 
+``conda`` may be rather slow, especially installing ``modin-all`` package so it's worth to run 
```

Comment:
```suggestion
``conda`` may be rather slow, especially installing ``modin-all`` package so it's worth trying to run 
```

Reply:
Also, do you think we should mention about omnisci package, which causes slowness of the installation (given the discussion in #4021)?